superset-frontend/src/SqlLab/components/SaveDatasetModal/index.tsx

```diff
@@ -151,6 +151,7 @@ export const SaveDatasetModal: FunctionComponent<SaveDatasetModalProps> = ({
   openWindow = true,
   formData = {},
 }) => {
+  console.log('FINDME SaveDatasetModal', datasource);
```

Comment:
remove this

Reply:
This was just debugging, it wasn't ready for review yet. I pushed up a "clean up" commit [`here`](https://github.com/apache/superset/pull/20975/commits/9f348bddec74224a808ac5ba88c7883e42f78c6a).